modules/osdk-cco-azure-enabling.adoc

```diff
@@ -0,0 +1,220 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/token_auth/osdk-cco-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="osdk-cco-azure-enabling_{context}"]
+= Enabling Operators to support CCO-based workflows with {azure-id}
+
+As an Operator author designing your project to run on Operator Lifecycle Manager (OLM), you can enable your Operator to authenticate against {azure-id}-enabled {product-title} clusters by customizing your project to support the Cloud Credential Operator (CCO).
+
+With this method, the Operator is responsible for creating the `CredentialsRequest` object, which means the Operator requires RBAC permission to create these objects. Then, the Operator must be able to read the resulting `Secret` object.
+
+[NOTE]
+====
+By default, pods related to the Operator deployment mount a `serviceAccountToken` volume so that the service account token can be referenced in the resulting `Secret` object.
+====
+
+.Prerequisities
+
+* {product-title} 4.14 or later
+* Cluster in {azure-id} mode
+* OLM-based Operator project
+
+.Procedure
+
+. Update your Operator project's `ClusterServiceVersion` (CSV) object:
+
+.. Ensure your Operator has RBAC permission to create `CredentialsRequests` objects:
++
+.Example `clusterPermissions` list
```

Comment:
You've inspired me to use collapsible for long [source,yaml] examples. Nice work.